keyboards/le_chiffre/keymaps/default/config.h

```diff
@@ -0,0 +1,4 @@
+#define IGNORE_MOD_TAP_INTERRUPT //helps with homerow mods
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Also, 
```suggestion
#pragma once

#define IGNORE_MOD_TAP_INTERRUPT //helps with homerow mods
```

Reply:
Thanks for the review